repose-aggregator/components/filters/uri-stripper-filter/src/main/resources/META-INF/schema/config/uri-stripper.xsd

```diff
@@ -65,14 +71,160 @@
                 </xs:documentation>
             </xs:annotation>
         </xs:attribute>
+    </xs:complexType>
+
+    <xs:complexType name="LinkResource">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    A specific ReST resource that contains one or more links which should have the stripped URI
+                    path segment re-inserted.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+
+        <xs:all>
+            <xs:element name="request" type="HttpMessage" minOccurs="0" maxOccurs="1"/>
+            <xs:element name="response" type="HttpMessage" minOccurs="0" maxOccurs="1"/>
+        </xs:all>
```

Comment:
Having `link-resource` and `response` in this order seems like there would be more boiler plate.  Unless there's an advantage to being able to pair a request/response in a link-resource??

e.g.
```
<link-resource>
    <response>
        <json link-mismatch-action="continue" token-index="0">$.link-a</json>
    </response>
</link-resource>
<link-resource>
    <response>
        <json link-mismatch-action="continue" token-index="1">$.link-b</json>
    </response>
</link-resource>
<link-resource>
    <response>
        <json link-mismatch-action="continue" token-index="5">$.link-c</json>
    </response>
</link-resource>
<link-resource>
    <response>
        <json link-mismatch-action="continue">$.foo.bar.baz.link</json>
    </response>
</link-resource>
```

versus:
```
<response>
    <link-response>
        <json link-mismatch-action="continue" token-index="0">$.link-a</json>
    </link-response>
    <link-response>
        <json link-mismatch-action="continue" token-index="1">$.link-b</json>
    </link-response>
    <link-response>
        <json link-mismatch-action="continue" token-index="5">$.link-c</json>
    </link-response>
    <link-response>
        <json link-mismatch-action="continue">$.foo.bar.baz.link</json>
    </link-response>
</response>
```